index.js

```diff
@@ -189,6 +190,10 @@ class ClinicDoctor extends events.EventEmitter {
     b.transform('brfs')
     const scriptFile = b.bundle()
 
+    if (/\bnode-clinic\b/i.test(process.env.NODE_DEBUG)) {
+      scriptFile.pipe(minifyStream({ sourceMap: false, mangle: false }))
+    }
```

Comment:
I would prefer a `settings.debug` option, so that we can do `clinic doctor --debug`  on the main project.